api/src/main/scala/za/co/absa/loginsvc/rest/provider/ad/ldap/ActiveDirectoryLDAPAuthenticationProvider.scala

```diff
@@ -92,15 +85,15 @@ class ActiveDirectoryLDAPAuthenticationProvider(config: ActiveDirectoryLDAPConfi
     def attempt(n: Int): Future[Authentication] = Future {
       Try(baseImplementation.authenticate(authentication)) match {
         case Success(auth) => auth
-        case Failure(ex) if isRetryableException(ex) && n <= attempts =>
```

Comment:
I think this was correct and we should count the retries as extra, so:
- `retryConfig = None`: should be: run once
- `retryConfig = LdapRetryConfig(2, delay)` should be: run once + 2 retries